webpack.config.js

```diff
@@ -53,6 +53,13 @@ module.exports = {
       {
         test: /\.(png|jpg)$/,
         use: 'file-loader'
+      },
+      {
+        test: /\.md$/,
+        use: [
+          { loader: 'html-loader' },
+          { loader: 'markdown-loader', options: { sanitize: true } }
```

Comment:
please check `sanitize` option description:
https://marked.js.org/#/USING_ADVANCED.md#options

Reply:
I've removed Webpack loaders. Now the application uses  `marked` and `DOMPurify` inside components.